francollamas/argentum-online
# Test Framework and Github Actions added

Github actions added to do the CI:
- build
- test
- check linter rules
- check format rules

Github action added to publish to Github Pages

Test Framework added: Vitest
